net/api/docu/client/docu_02httpclient_01_test.cc

```diff
@@ -38,7 +38,7 @@ TEST(Docu, HTTPClient01A) {
 HTTP(FLAGS_docu_net_client_port_01).ResetAllHandlers();
 HTTP(FLAGS_docu_net_client_port_01).Register("/ok", [](Request r) { r("OK"); });
 #if 1
-EXPECT_EQ("OK", HTTP(GET(Printf("localhost:%d/ok", FLAGS_docu_net_client_port_01))).body);
+EXPECT_EQ("OK", HTTP(GET(Printf("http://localhost:%d/ok", FLAGS_docu_net_client_port_01))).body);
```

Comment:
Could you please also make this change in the other clause of this `#if`? It does to the `README.md` documentation, would be nice to keep it clean.

(And run `make docu` from the top-level directory of the repository to re-generate `README.md`).